internal/controller/storagemgr/storagedriver/storagehandler.go

```diff
@@ -236,8 +235,7 @@ func (handler StorageHandler) processAsyncPostRequest(writer http.ResponseWriter
 	value, err := handler.newCommandValue(resourceName, reading, readingType)
 	if err != nil {
 		log.Error(
-			fmt.Sprintf("Incoming reading ignored. Unable to create Command Value for Device=%s Command=%s: %s",
-				deviceName, resourceName, err.Error()))
+			fmt.Sprintf("Incoming reading ignored. Unable to create Command Value for Device=%s Command=%s: %s", strings.ReplaceAll(fmt.Sprintf("%#v", deviceName), "\"", ""), strings.ReplaceAll(fmt.Sprintf("%#v", resourceName), "\"", ""), err.Error())) // lgtm [go/log-injection]
```

Comment:
This line is too long. How about using 2 lines?

Reply:
See above